test/perf/benchmarks/Engine.Compiler.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#if NET6_0
```

Comment:
Why just NET6, it would be good if we can compile and run against NET to compare perf between 7.1 and 7.2 right?

Reply:
The test uses the internal API directly. 7.1 and 7.2 doesn't have the internal visible attribute defined for powershell-perf.dll